Keyword only args causes RequestMiddleware to fail on Django app load

#### Environment details

  - OS type and version: macOS version 11.0.1
  - Python version: Python 3.8.2
  - pip version: pip 20.2.4
  - `google-cloud-logging` version: 2.0.1 and 2.0.0

#### Steps to reproduce

  1. Add `google.cloud.logging_v2.handlers.middleware.RequestMiddleware` to a `MIDDLEWARE` list in django app settings
  2. Make an API request, e.g. during a test 

#### Stack trace
```
self = <django.test.client.ClientHandler object at 0x124a5f850>

    def load_middleware(self):
        """
        Populate middleware lists from settings.MIDDLEWARE.

        Must be called after the environment is fixed (see __call__ in subclasses).
        """
        self._view_middleware = []
        self._template_response_middleware = []
        self._exception_middleware = []

        handler = convert_exception_to_response(self._get_response)
        for middleware_path in reversed(settings.MIDDLEWARE):
            middleware = import_string(middleware_path)
            try:
>               mw_instance = middleware(handler)
E               TypeError: __init__() takes 1 positional argument but 2 were given
```

#### Extra Info
It looks like django [doesn't support keyword only arguments](https://github.com/django/django/blob/d8dfff2ab0edf7a1ca5255eccf45c447b2f9d57e/django/core/handlers/base.py#L58) when loading middleware (the link is pointing to master, but it's the same for old versions). If I locally do the following, I can subclass the `RequestMiddleware` and remove the keyword only args and the whole issue goes away

--- a/google/cloud/logging_v2/handlers/middleware/request.py
+++ b/google/cloud/logging_v2/handlers/middleware/request.py
@@ -42,7 +42,7 @@ except ImportError:  # pragma: NO COVER
 class RequestMiddleware(MiddlewareMixin):
     """Saves the request in thread local"""
 
-    def __init__(self, get_response=None):
+    def __init__(self, get_response):
         self.get_response = get_response
 
     def process_request(self, request):

--- a/google/cloud/logging_v2/handlers/middleware/request.py
+++ b/google/cloud/logging_v2/handlers/middleware/request.py
@@ -42,7 +42,7 @@ except ImportError:  # pragma: NO COVER
 class RequestMiddleware(MiddlewareMixin):
     """Saves the request in thread local"""
 
-    def __init__(self, *, get_response=None):
+    def __init__(self, get_response=None):
         self.get_response = get_response
 
     def process_request(self, request):

--- a/tests/unit/handlers/middleware/test_request.py
+++ b/tests/unit/handlers/middleware/test_request.py
@@ -54,6 +54,16 @@ class TestRequestMiddleware(DjangoBase):
         django_request = request._get_django_request()
         self.assertEqual(django_request, mock_request)
 
+    def test_can_instantiate_middleware_without_kwargs(self):
+        handler = mock.Mock()
+        middleware = self._make_one(handler)
+        self.assertEqual(middleware.get_response, handler)
+
+    def test_can_instantiate_middleware_with_kwargs(self):
+        handler = mock.Mock()
+        middleware = self._make_one(get_response=handler)
+        self.assertEqual(middleware.get_response, handler)
+
 
 class Test__get_django_request(DjangoBase):
     @staticmethod

--- a/tests/unit/handlers/middleware/test_request.py
+++ b/tests/unit/handlers/middleware/test_request.py
@@ -41,6 +41,9 @@ class TestRequestMiddleware(DjangoBase):
         return request.RequestMiddleware
 
     def _make_one(self, *args, **kw):
+        if not args and "get_response" not in kw:
+            kw["get_response"] = None
+
         return self._get_target_class()(*args, **kw)
 
     def test_process_request(self):

--- a/tests/unit/handlers/test__helpers.py
+++ b/tests/unit/handlers/test__helpers.py
@@ -87,7 +87,7 @@ class Test_get_trace_id_from_django(unittest.TestCase):
 
         django_request = RequestFactory().get("/")
 
-        middleware = request.RequestMiddleware()
+        middleware = request.RequestMiddleware(None)
         middleware.process_request(django_request)
         trace_id = self._call_fut()
         self.assertIsNone(trace_id)
@@ -104,7 +104,7 @@ class Test_get_trace_id_from_django(unittest.TestCase):
             "/", **{django_trace_header: django_trace_id}
         )
 
-        middleware = request.RequestMiddleware()
+        middleware = request.RequestMiddleware(None)
         middleware.process_request(django_request)
         trace_id = self._call_fut()
 

